examples/quickstart_etl/README.md

```diff
@@ -0,0 +1,208 @@
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201304011-0c62ad06-36d8-4984-9278-b3b670254da3.png">
+</p>
+
+
+<TODO: explain word cloud plot 👇 . need to find a cloud friendly way to view the plot>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201312310-44fb053c-9352-498d-9c23-ab4d7be6afec.png">
+</p>
+
+## Step 3: Configuring data pipelines
+
+You've got your first couple of assets generated. Let's add one more data source to the pipeline: GitHub.
+
+Navigate to the `github` asset group and click `Materialize all`. When you try to materialize assets in this group, you may encounter the following error:
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200670846-625d1196-c421-4906-82fc-1d045c51e158.png">
+</p>
+
+<TODO: explain config and env vars>
+
+### Step 3.1: Setting up GitHub Token
+
+#### 1) Generating GitHub Personal Access Token
+
+<TODO: how to generate github PAT>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201305411-76f0c39f-887b-4162-a15e-e41fba001320.png">
+</p>
+
+#### 2) Adding the token as an environment variable
+
+1. Add the token to your environment: `export MY_GITHUB_TOKEN="blahblahblah-todo-put-something-looks-like-a-token-here" inside either your `.bash_profile`, `.bashrc`, or `.zshrc`. *Important: You will need to `source` your bash profile (or restart the terminal) afterwards.
+2. Verify you see your token: Open a new terminal and run `echo $MY_GITHUB_TOKEN`.
+3. All set! Start `dagit` again and now you should be able to materialize all the `github` assets using your own GitHub token.
+
+#### 3) (Dagster Cloud) Environment Variable page
+
+<TODO: add env var to dagster cloud>
+
+### Step 3.2: (Optional) Sampling data fetching
+
+<details><summary>Expand to view</summary>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200671284-c716efa5-1a0f-4a54-a79b-5247f1589374.png">
+</p>
+
+```yaml
+ops:
+  hackernews_topstories:
+    config:
+      sample_size: 100
+```
+
+</details>
+
+
+### Step 3.3: (Optional) Filtering stories by keyword
+
+
+<details><summary>Expand to view</summary>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200671547-11073c00-fe35-4296-9457-b41a8e8ab395.png">
+</p>
+
+```yaml
+ops:
+  hackernews_stories_by_date:
+    config:
+      keyword: twitter
+```
+
+</details>
+
+
+## Step 4: Turning on the daily schedule
+
+Go to <TODO: schedule page>. Turn on the schedule.
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200672130-dd2f0cdc-e767-448b-9ed0-b05eaa4a9c69.png">
+</p>
+
+Congratulations 🎉 You now have a daily metrics report running in production!
+
+## Learn more
+
+<TODO: links to other post NUX guides>
+
+### Adding new Python dependencies
```

Comment:
this is because users may wonder how to do it. it's not always obvious that `setup.py` is the recommended place - esp helpful in the serverless case because we control the docker image and it's installing dependencies based on `setup.py`

Reply:
gotcha, makes sense